error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/ReactorRulesTestInput.java

```diff
@@ -389,8 +389,10 @@ StepVerifier.FirstStep<Integer> testStepVerifierFromFlux() {
     return StepVerifier.create(Flux.just(1));
   }
 
-  StepVerifier.Step<Integer> testStepVerifierStepExpectNextEmpty() {
-    return StepVerifier.create(Mono.just(0)).expectNext();
+  ImmutableSet<StepVerifier.Step<Integer>> testStepVerifierStepIdentity() {
+    return ImmutableSet.of(
+        StepVerifier.create(Mono.just(0)).expectNext(),
+        StepVerifier.create(Mono.just(0)).expectNextCount(0L));
   }
```

Comment:
(a) All the other examples contain / start with `Mono.just(1)`. Let's simply also apply this here. :+1:
(b) Cases in which there are multiple input cases, they'll be represented by different values. More concretely, I suggest this:
```suggestion
  ImmutableSet<StepVerifier.Step<Integer>> testStepVerifierStepIdentity() {
    return ImmutableSet.of(
        StepVerifier.create(Mono.just(1)).expectNext(),
        StepVerifier.create(Mono.just(2)).expectNextCount(0L));
  }
```